modules/tinymce/src/plugins/lists/main/ts/actions/ToggleList.ts

```diff
@@ -230,13 +235,14 @@ const updateList = function (editor: Editor, list, listName, detail) {
 };
 
 const toggleMultipleLists = function (editor, parentList, lists, listName, detail) {
-  if (parentList.nodeName === listName && !hasListStyleDetail(detail)) {
+  if (NodeType.isListNode(parentList) && parentList.nodeName === listName && !hasListStyleDetail(detail)) {
     flattenListSelection(editor);
   } else {
     applyList(editor, listName, detail);
     const bookmark = Bookmark.createBookmark(editor.selection.getRng(true));
+    const allLists = NodeType.isListNode(parentList) ? [ parentList, ...lists ] : [ ...lists ];
```

Comment:
Worth pulling `NodeType.isListNode(parentList)` into a var since it's used in both branches? It's only a minor thing...